<launch>
	<arg name="map"/>
	
	<node name="map_loader" pkg="map_server" type="map_server" args="$(arg map)"/>
	
	<node name="rviz1" pkg="rviz" type="rviz" args="-d $(find nav_goals)/rviz/config.rviz"/>
	
	<node name="generate_goals_node" pkg="nav_goals" type="generate_goals_node" output="screen">
		<param name="goals_file_name" value="$(find nav_goals)/goals/goals.xml"/>
		<param name="is_add_mode" value="false"/>
	</node>
	
</launch>
